apps/OboeTester/app/src/main/cpp/NativeAudioContext.cpp

```diff
@@ -251,6 +258,20 @@ oboe::Result ActivityContext::start() {
         dataThread = new std::thread(threadCallback, this);
     }
 
+#ifdef DEBUG_CLOSE_RACE
```

Comment:
If we are leaving this in, please add a comment explaining exactly what this directive is used for.

Reply:
Done.